Tools/wasm/config.site-wasm32-emscripten

```diff
@@ -34,6 +34,7 @@ ac_cv_lib_bz2_BZ2_bzCompress=no
 # https://github.com/pyodide/pyodide/blob/main/cpython/pyconfig.undefs.h
 
 ac_cv_func_epoll=no
```

Comment:
Looking at 994f90c0772612780361e1dc5fa5223dce22f70a, this line should be removed?

Reply:
Yes, good catch